### ISSUE TITLE
[Bug] When I open the Interstellar page it just turns into a NASA page

### ISSUE THREAD

**Issue by biggiecheeez** [UseInterstellar/Interstellar#711]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**
If applicable, could you add screenshots to help explain your problem?

**Device Information (please complete the following information):**

- Device: [Ex. Chromebook, iPad, etc.]
- OS [Ex. ChromeOS, Windows, iOS
- Browser [Ex. Chrome, Safari]

**Additional context**
Please feel free to add any other context about the problem here.

> [!IMPORTANT]
> Do not post your personal information here this includes but is not limited to email addresses, names, and ages.

**Comment by IncognitoTGT** [comment]
yeah he explained the issue, probably one of the best OSS issue i've seen since i started developing stuff.

**Comment by biggiecheeez** [comment]
aight thanks

**Comment by IncognitoTGT** [comment]
bro does NOT understand sarcasm
